docs/logs/getting-started/README.md

```diff
@@ -61,6 +61,12 @@ instance, which is then used to do the logging. The log is sent to the
 `OpenTelemetryLoggerProvider`, which is configured to export logs to
 `ConsoleExporter`. `ConsoleExporter` simply displays it on the console.
 
+If you are using ASP.NET Core applications, then adding logging providers
```

Comment:
Consider something like:

In a real application, the `ILogger` instance might be obtained using a very different way, here are some useful links:
* Logging in ASP.NET Core

Reply:
And maybe link to the example, e.g. https://github.com/open-telemetry/opentelemetry-dotnet/blob/6b0883eb224fb143b14e527cc622eeb584c58770/examples/AspNetCore/Program.cs#L89

Reply:
this sounds better.! let me tweak further.